lib/page/addPatient.dart

```diff
@@ -72,7 +72,8 @@ class _AddPatientPageState extends State<AddPatientPage> {
         'state': 'Pre-Operation',
         'weight': 0,
         'height': 0,
-        'oldWeight': 0
+        'oldWeight': 0,
+        'latestStateChange': latestStateChange,
```

Comment:
We will store latestStateChange since add patient ?

Reply:
@PormpimonPang  It was my intention at first. and, I forgot to store it. Anyway it will be set to the time that user is created. I think it make sense since when a user is created => the patient has pre-op state at that time.
Moreover, it helps us get rid of that null exception on the pre-op users when going into patientDetailPage